lib/src/component/aria_mixin.dart

```diff
@@ -179,10 +252,22 @@ abstract class _$AriaPropsMixin {
   /// is no supported operation. In addition to setting the aria-dropeffect attribute, authors SHOULD
   /// show a visual indication of potential drop targets.
   ///
-  /// See: <http://www.w3.org/TR/wai-aria/states_and_properties#aria-dropeffect>
+  /// See: <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-dropeffect>
+  /// TODO: deprecate
   @Accessor(key: 'aria-dropeffect')
   dynamic dropeffect;
 
+  /// Identifies the element that provides an error message for that object
+  ///
+  /// When there is a user-created error, you want to let them know it exists and tell them how to fix it. 
+  /// There are two attributes you need to use: set aria-invalid="true" to define the object as being in 
+  /// an error state, then add the aria-errormessage attribute with the value being the id of the element 
+  /// containing the error message text for that object
+  ///
+  /// See: <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage>
+  @Accessor(key: 'aria-errormessage')
+  String error;
```

Comment:
Is there a reason this is `error` instead of `errormessage`?
```suggestion
  String errormessage;
```